roles/dosomething.jenkins/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+- name: Add Jenkins Repo Key
+  become: yes
+  apt_key: url=https://jenkins-ci.org/debian/jenkins-ci.org.key
```

Comment:
It'd be great to add the [key](http://docs.ansible.com/ansible/apt_key_module.html) argument:
> `id` identifier of key. Including this allows check mode to correctly report the changed state.
You can find the key using `apt-key list` command.